contrib/vagrant/provision-util.sh

```diff
@@ -399,3 +400,36 @@ os::provision::wait-for-node-config() {
   os::provision::wait-for-condition "${msg}" "${condition}" \
     "${OS_WAIT_FOREVER}"
 }
+
+# Enable overlayfs for dind if it can be tested to work.
+os::provision::enable-overlay-storage() {
+  local storage_dir=${1:-/var/lib/docker}
+
+  local msg="WARNING: Unable to enable overlay storage for docker-in-docker"
+
+  if grep -q overlay /proc/filesystems; then
```

Comment:
> I don't have overlay in my /proc/filesystems on Fedora 23 by default. Perhaps 'modprobe overlay || true' before checking if overlay exists?

Nevermind on this one, it doesn't happen inside a vagrant VM with 'OPENSHIFT_DEV_DIND=1 vagrant up' but then again it's not supposed to.